### PR TITLE
improvement(manager): save Scylla manager version in Argus

### PR DESCRIPTION
**Pull request by juliayakovlev**
Collect and save in Argus Scylla manager version

Screenshot from 2024-06-03 18-12-08

### Testing

- [x] [longevity-100gb-4h](https://argus.scylladb.com/test/657bf1f3-69a5-4c36-bdbf-bf17718f7087/runs?additionalRuns[]=ea7e5f7b-34ec-4e41-a168-51024930ebfe)

### PR pre-checks (self review)

- [ ] I added the relevant `backport` labels
- [ ] I didn't leave commented-out/debugging code

### Reminders

- Add New configuration option and document them (in `sdcm/sct_config.py`)
- Add unit tests to cover my changes (under `unit-test/` folder)
- Update the Readme/doc folder relevant to this change (if needed)

### CODE REVIEW

File: sdcm/tester.py [hunk 1]
```diff
@@ -502,6 +503,20 @@ def argus_collect_packages(self):
         except Exception:  # pylint: disable=broad-except
             self.log.error("Unable to collect package versions for Argus - skipping...", exc_info=True)
 
+    def argus_collect_manager_version(self):
+        if self.monitors:
```
Comment (fruch) [inline sdcm/tester.py:507]: you might want to limit it, if we enable manager or not, we have a flag for that in configuration

File: sdcm/tester.py [hunk 2]
```diff
@@ -817,10 +832,10 @@ def is_encrypt_keys_needed(self):
 
     def prepare_kms_host(self) -> None:
         if (self.params.is_enterprise and ComparableScyllaVersion(self.params.scylla_version) >= '2023.1.3'
-            and self.params.get('cluster_backend') == 'aws'
-            and not self.params.get('scylla_encryption_options')
-            and self.params.get("db_type") != "mixed_scylla"  # oracle probably doesn't support KMS
-            ):
+                and self.params.get('cluster_backend') == 'aws'
```
Comment (fruch) [inline sdcm/tester.py:835]: pycharm keep insisting on this one....

File: sdcm/tester.py (hunk 1 above)
Comment (juliayakovlev) [reply sdcm/tester.py:507]: Do you mean "n_monitor_nodes"?

File: sdcm/tester.py (hunk 2 above)
Comment (juliayakovlev) [reply sdcm/tester.py:835]: it is changing by pylint. It is changed every time when I run pylint

File: sdcm/tester.py (hunk 1 above)
Comment (soyacz) [reply sdcm/tester.py:507]: `use_mgmt` param

File: sdcm/tester.py (hunk 2 above)
Comment (vponomaryov) [reply sdcm/tester.py:835]: Use `git add -p %file%` approach to commit only part of the changes in a file.

Comment (soyacz) [reply sdcm/tester.py]: I'd just merge it so we never see it again - no need to waste time on thinking about it.

**Review by fruch — APPROVED** [review]
LGTM

**Pull request merged**